apps/prairielearn/src/lib/cache.ts

```diff
@@ -1,14 +1,20 @@
-import redis = require('redis');
+import { Redis } from 'ioredis';
 import { LRUCache } from 'lru-cache';
 import { logger } from '@prairielearn/logger';
 import * as Sentry from '@prairielearn/sentry';
 
 import { config } from './config';
 
+const CACHE_KEY_PREFIX = 'prairielearn-cache:';
+
 let cacheEnabled = false;
 let cacheType: 'redis' | 'memory' | 'none';
 let cache: LRUCache<string, string>;
-let client: redis.RedisClientType;
+let client: Redis;
+
+function cacheKey(key: string): string {
+  return CACHE_KEY_PREFIX + key;
+}
```

Comment:
I realized while testing locally that we should probably scope our cache keys so that we can safely delete *just* the values from the cache, which will be relevant if we start using the Redis instance for other purposes. There's currently no pressing reason to do this, just seemed like a good idea. This will effectively wipe the cache when this is deployed.

Reply:
Excellent point!